app/src/composables/use-flows.ts

```diff
@@ -183,6 +189,7 @@ export function useFlows(options: UseFlowsOptions) {
 	function provideRunManualFlow() {
 		provide(runManualFlowSymbol, {
 			runManualFlow,
+			manualFlows,
```

Comment:
This was added to the injection in order to be used within an interface in order to figure out whether a flow is currently loading or not.

This was not a problem in `flow-sidebar-detail` as the manualFlows object was passed directly as a prop to that component from the item route: https://github.com/directus/directus/blob/2c67450512647595b71fb0fbbca83fc4dd52d797/app/src/views/private/components/flow-sidebar-detail.vue#L7

If we go with this pattern than we can also remove manualFlows from the exports for `useFlows` and just receive it through the injection rather than as a prop in flow-sidebar-detail 🤔 

Unless there is a better way to pass manualFlows from the item route to the child field?